src/DisplayTouristCard.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import TouristInfoCards from "./TouristInfoCard";
+function DisplayTouristCard() {
```

Comment:
```suggestion
function DisplayTouristCards() {
```
And make this one plural as it displays multiple cards - sorry to be so picky 😛